src/org/traccar/notification/EventForwarder.java

```diff
@@ -20,6 +20,7 @@
 import java.util.Arrays;
 import java.util.List;
 
+import org.apache.commons.lang3.StringUtils;
```

Comment:
Please don't use this. There is no explicit dependency for this library and I don't see any reason for adding dependency just to check for empty string.

Reply:
Done.

I would like to add my opinion on this,

you need to do 2 checks, null and isEmpty, if you search on the project you will find several occurrences of this kind of checks, it could be simpler using apis.

You could consider to re-add those library apis in the future, they are almost used everywhere, so you will end having them inside your project frequently, personally i don't see why this should be avoided, if you don't start to use them you will never use them and that could lead to repeatedly reinvent the wheel on your project or to start an implementation of your own set of utilities for simple operations; both solutions could be highly ineffective. Specially when you have around 40 Mb of "ready to use" code on your project.